content/how-to/dedicated-hosting/delete-vpn-user-myrackspace-portal/index.md

```diff
@@ -19,16 +19,23 @@ This article describes how to delete a VPN user by using a ticket template.
 1. Log in to the [MyRackspace Portal](https://login.rackspace.com) by using your
    username and password.
 
-2. In the top navigation bar, click **Select a Product > Rackspace Dedicated**.
+   <img width="409" alt="logincredentials" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn1.png">
 
-3. Select **Tickets** > **Create New Ticket**.
+2. In the top navigation bar, click **Support > Support Center**.
 
-4. On the **Create New Ticket** page, click the **Subject** text field, and in
-   the drop-down menu, select **VPN User Management - add, reset, delete VPN user(s) on firewall(s)**.
+   <img width="941" alt="selectsupport" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn2.png"> 
 
-   {{<image src="image003.jpeg" alt="" title="">}}
+3. From the **Common Request** menu select **VPN - add, reset, delete VPN user(s) on firewalls** as shown in following image.
 
-5. Check the box next to **Delete VPN user on firewall**.
+   <img width="830" alt="selectdeletevpn" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn3.png">
+
+4. You can view **Issue Details** section by selecting **VPN - add, reset, delete VPN user(s) on firewalls** as shown in following image.
```

Comment:
add `the` before issues details.. for example `You can view the **Issue Details** section`

Reply:
Done